lacework-agent/Chart.yaml

```diff
@@ -14,4 +14,4 @@ maintainers:
 - email: info@lacework.net
   name: lacework-support
 name: lacework-agent
-version: 6.5.0
```

Comment:
I started a Slack thread about whether the previous version of the file should have been updated to 6.5.2. Did we miss it last release?

Reply:
It looks like we did miss it.